cluster/vagrant/setup_kubernetes_master.sh

```diff
@@ -28,7 +28,14 @@ systemctl enable cockpit.socket && systemctl start cockpit.socket
 rm -rf /var/lib/kubelet
 
 # Create the master
-kubeadm init --pod-network-cidr=10.244.0.0/16 --token abcdef.1234567890123456
+kubeadm init --config <(cat <<EOF)
+apiVersion: kubeadm.k8s.io/v1alpha1
+kind: MasterConfiguration
+apiServerExtraArgs:
+  runtime-config: admissionregistration.k8s.io/v1alpha1
```

Comment:
bummer, I didn't realize this initializer stuff was still a non-default alpha feature. 

@stu-gott If this feature isn't toggled on, does that mean users just won't be able to use presets, or will it prevent VMs from being scheduled entirely?

Reply:
VM's will continue to work. If presets are used a weird race condition will ensue: Since the initializer code will be present, it will still act normally--except it won't get a head start before other cluster resources (e.g. virt-controller) also start processing the VM. This is obviously undesirable.

If we are forced down a path where we need to deal with this without using Kubernetes initializer mechanism, the solution will be for virt-controller to manually inspect the VM to ensure the VMPreset initializer is given a chance to operate before it acts.

Reply:
... or switch to a different approach entirely of course.

Reply:
it seems like webhooks which allow mutating CRDs is the new thing in k8s. They were not even there when initializers became alpha, but there seems to be an agreement now that webhooks are much simpler and they are now already beta in 1.9.